var/spack/repos/builtin/packages/py-kitchen/package.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyKitchen(PythonPackage):
+    """Kitchen contains a cornucopia of useful code
+    """
+
+    homepage = "https://fedorahosted.org/kitchen"
+    version('1.2.6', sha256='b84cf582f1bd1556b60ebc7370b9d331eb9247b6b070ce89dfe959cba2c0b03c')
```

Comment:
@Sinan81 This version appears twice, could you remove one of them?

Reply:
Ooops. It seems I overlooked it.